igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/service/TravisResultConverter.groovy

```diff
@@ -25,6 +25,9 @@ import groovy.util.logging.Slf4j
 class TravisResultConverter {
     static Result getResultFromTravisState(String state) {
         switch (state) {
+            case "created":
+                return Result.NOT_BUILT
+                break
             case "started":
                 return Result.BUILDING
                 break
```

Comment:
I still think this line https://github.com/spinnaker/igor/pull/80/files#diff-6e2e5f43c3a3dc4ecf7a723feb49ee26R47 should throw an exception, since it's a failure state for igor.